org.eclipse.jdt.ls.core/src/org/eclipse/jdt/ls/core/internal/JDTUtils.java

```diff
@@ -425,6 +418,17 @@ public static String getFileURI(IResource resource) {
 		return ResourceUtils.fixURI(resource.getRawLocationURI());
 	}
 
+	public static String getFileURI(IClassFile classFile) {
+		String packageName = classFile.getParent().getElementName();

```

Comment:
This just refers to the existing JDTUtils#toLocation method. What does the standard JDT Scheme look like?